uncertainty_baselines/datasets/datasets.py

```diff
@@ -98,13 +98,13 @@ def get(
   logging.info(
       'Building dataset %s with additional kwargs:\n%s',
       dataset_name,
-      json.dumps(hyperparameters, indent=2, sort_keys=True))
+      json.dumps(hyperparameters, indent=2, sort_keys=True),
+  )
```

Comment:
we've generally avoided the hanging parentheses in the codebase, the other lint edits are fine (we haven't been super consistent on how to do indentations for vertical parameter/arg lists)